### PR TITLE
Feature/billing worklogs

### PR DESCRIPTION
**Pull request by tuj**
* Changed from working with issues to registering worklogs to invoice entries.

### CODE REVIEW

File: src/Service/AbstractJiraService.php [hunk 1]
```diff
@@ -204,6 +204,46 @@ public function getBoard($boardId)
...
+     *
+     * @param $projectId
+     * @param string $from
+     * @param string $to
+     *
+     * @return mixed
+     */
+    public function getProjectWorklogs($projectId, $from = '2000-01-01', $to = '3000-01-01')
+    {
+        $worklogs = $this->post('rest/tempo-timesheets/4/worklogs/search', [
```
Comment (rimi-itk) [inline src/Service/AbstractJiraService.php:218]: What is `4` here? An API version?

File: bundles/Billing/Resources/assets/css/react-datepicker.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.react-datepicker-wrapper {
```
Comment (rimi-itk) [inline bundles/Billing/Resources/assets/css/react-datepicker.css]: Maybe use `scss` and nesting?

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good. However, a few comments should be addressed.

File: src/Service/AbstractJiraService.php (hunk 1 above)
Comment (tuj) [reply src/Service/AbstractJiraService.php:218]: Yes.

**Review by rimi-itk — APPROVED** [review]
Hooray!

**Pull request merged**